centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl

```diff
@@ -337,7 +340,21 @@
       </if>
       <else-if type="article-journal article-magazine">
         <group delimiter="" font-style="normal">
-          <choose>
+          <group delimiter="/">
+            <group delimiter=" ">
+              <text term="volume" form="short" suffix="."/>
+              <text variable="volume"/>
+            </group>
+            <group delimiter=" ">
+              <text variable="number-of-volumes"/>
+              <choose>
+                <if variable="volume" match="none">
+                  <text term="volume" form="short" suffix="."/>
+                </if>
+              </choose>
+            </group>
+          </group>
+          <!--<choose>
```

Comment:
The Travis-CI tests currently fail because the "volume" macro is now no longer used, so it should be deleted.

In general, it's also better to just delete unused CSL code instead of commenting it out.